Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1021,6 +1024,12 @@ eval BackPressed state = do
                       _ <- pure $ removeAllPolylines ""
                       _ <- pure $ updateLocalStage SearchLocationModel
                       continue state{props{defaultPickUpPoint = "", rideRequestFlow = false, currentStage = SearchLocationModel, searchId = "", isSource = Just false,isSearchLocation = SearchLocation,isSharedLocationFlow = false},data{polygonCoordinates = "", nearByPickUpPoints = []}}
+    ConfirmingDropLocation -> do
+                      void $ pure $ performHapticFeedback unit
+                      _ <- pure $ exitLocateOnMap ""
+                      _ <- pure $ removeAllPolylines ""
+                      _ <- pure $ updateLocalStage SearchLocationModel
+                      continue state{props{defaultPickUpPoint = "", defaultDropOfPoint = "",rideRequestFlow = false, currentStage = SearchLocationModel, searchId = "", isSource = Just false,isSearchLocation = SearchLocation,isSharedLocationFlow = false},data{polygonCoordinates = "", nearByPickUpPoints = [],dropPolygonCoordinates = "",nearByDropOfPoints = []}}
```

Comment:
create a function for these line of code and invoke that function from both ConfirmingDropLocation and ConfirmingLocation